### PR TITLE
Meta improvements

### PR DESCRIPTION
**Pull request by athena413**
#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md

# General Documentation

### Intent of your Pull Request

Makes a ton of changes to YogsMeta. 

Includes but is not limited to:

Gives the Janitor his janitorial cart

Gives the Roboticists anaesthetic

Gives Toxins an electrolyzer and some water vapor

Gives Medbay 2 surgery tables

All changes written on pastebin https://pastebin.com/vKPwxDJD

### Why is this change good for the game?

meta is better now

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Meta has been changed a ton, hopefully for the better

### What should players be aware of when it comes to the changes your PR is implementing?

Pretty much the entire map has been improved - it's not it's "best" but there's a good few serious upgrades

### What general grouping does this PR fall under? 

Mapping - the whole map

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

N/A

# Changelog

:cl:  
tweak: Tons of changes to the map
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
This might be on hold for a while because icemeta seems to be moving again(?).
Still, with the pastebin the changes can be put in easily.

**Comment by Cartlord** [comment]
All looks good. Would it be OK to request that Atmos get a hazmat suit for use with their turbine?

**Comment by patpol4** [comment]
Will need to be implemented on https://github.com/yogstation13/Yogstation/pull/10535 again, keep the pastebin please

**Review by patpol4 — APPROVED** [review]
Did some minor tweaks like moving all head spawns to meeting room, unkinked some wires, undedinged the library, Citrus gets a bed instead of Autumn, spade in garden, removed sunglasses from lawyer office, actually replaced the sunglasses in showroom with cheap sunglasses. All else seems fine, really needed touchups. Thanks
